### PR TITLE
CVE-2009-3095

### PR DESCRIPTION
**Pull request by Nitepone**
- Set curated to true
- Updated description

### CODE REVIEW

File: cves/CVE-2009-3095.yml [hunk 1]
```diff
@@ -32,7 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+        This vulnerability allowed authorized attacks to send commands to an
+        FTP server due to inadequate input validation.
```
Comment (Pearse4004) [inline cves/CVE-2009-3095.yml:37]: Describe how the vulnerability allowed authorized attacks to send commands

Comment (Nitepone) [reply cves/CVE-2009-3095.yml:37]: Updated commit better describes how attacks are created

File: cves/CVE-2009-3095.yml [hunk 2]
```diff
@@ -54,7 +57,7 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: 404947824a978c89589d0fa59ada0ee1531bb147
-     note:
+     note: 'A check was added to throw out strings with special characters'
    - commit:
      note:
 vccs:
```
Comment (aldenreed) [inline cves/CVE-2009-3095.yml:63]: I would recommend adding a note that says what was changed in commit a83b21763cd2fbfd97bf5cbb5f01ea4d7e7ac3c3.

File: cves/CVE-2009-3095.yml [hunk 3]
```diff
@@ -202,4 +207,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The mastake that lead to this vulnerability is a simple coding mistake.
```
Comment (aldenreed) [inline cves/CVE-2009-3095.yml]: *mistake

**Comment by aldenreed** [comment]
I will give this vulnerability 6 upvotes.

**Review by allisonsuarez — COMMENTED** [review]
This could be a complex inputs issue?

**Comment by Nitepone** [comment]
> This could be a complex inputs issue?

I considered this when choosing the issue. The problem is that the input isn't really that complicated, it is just a single string. I ended up choosing to not call it a complex inputs issue as I feel there are far more clear cut examples.

**Pull request merged**